README.md

```diff
@@ -699,6 +700,42 @@ export default () => (
 );
 ```
 
+### Corporate Contact
+
+```jsx
+import React from 'react';
+import { CorporateContactJsonLd } from 'next-seo';
+
+export default () => (
+  <>
+    <h1>Corporate Contact JSON-LD</h1>
+    <CorporateContactJsonLd
+      data-testid="corporate-contact"
```

Comment:
I don't think the `data-testid="corporate-contact"` is actually used anywhere? 

If not could remove it from here and anywhere else it is.

Reply:
Ha! I was testing using `data-testid` for cypress testing instead of using position index and forgot to remove it after.